### PR TITLE
Another VMSS autoscale template with script extension

### PR DESCRIPTION
**Pull request by madhana**
VMSS template showing autoscale running a Apache/PHP app

### CODE REVIEW

File: 201-vmss-lap-autoscale/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+### Autoscale a VM Scale Set running a Ubuntu/Apache/PHP app ###
```
Comment (singhkay) [inline 201-vmss-lap-autoscale/README.md]: I think you meant to call this folder "lamp" instead of lap :)

File: 201-vmss-lap-autoscale/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Deploy the scale set with an instance count of 1 
+- After it is deployed look at the resource group public IP address resource (in portal or resources explorer). Get the IP or domain name.
+- Browse to the website (port 80), which shows the current backend VM name.
+- Hit the "Do work" button with an iteration count of say 300 (represents seconds of max CPU).
+- After a few minutes the scale set capacity will increase, and refreshing the browser and going to the home page a few times will show additional backend VM name(s).
+- You can increase the work by connecting to more backend websites, or decrease by letting the iterations time-out, in which case the scale set will scale down - hence after about 10 minutes the capacity should be back down to 1.
+
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2F201-vmss-lap-autoscale%2Fazuredeploy.json" target="_blank">
```
Comment (singhkay) [inline 201-vmss-lap-autoscale/README.md]: This will change as well once the folder name is updated

File: 201-vmss-lap-autoscale/metadata.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Deploy a VM Scale Set with Linux VMs and Auto Scale configured",
```
Comment (singhkay) [inline 201-vmss-lap-autoscale/metadata.json]: Title and description need to be updated. Currently it is the same description as other vmss autoscale template.

File: 201-vmss-lap-autoscale/README.md (hunk 1 above)
Comment (madhana) [inline 201-vmss-lap-autoscale/README.md]: no, it is LAP. it doesn't have MySQL :)

**Pull request merged**